paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/EnableInstantDebitsSettingsDefinition.kt

```diff
@@ -12,6 +12,12 @@ internal object EnableInstantDebitsSettingsDefinition : BooleanSettingsDefinitio
     override fun configure(value: Boolean, checkoutRequestBuilder: CheckoutRequest.Builder) {
         // Hijacking this method to synchronize the setting with the feature flag
         FeatureFlags.instantDebits.setEnabled(value)
+
+        if (value) {
+            checkoutRequestBuilder.supportedPaymentMethods(
+                supportedPaymentMethods = listOf("card", "link")
+            )
```

Comment:
Make sure `us_bank_account` doesn’t show up in the list, so we can indeed show Instant Debits.

Reply:
I don't think we want to do this. We can use the `SupportedPaymentMethodsSettingsDefinition` instead.

Reply:
Oh, didn’t know that existed. Updating now!